data/card_definitions.json

```diff
@@ -15061,6 +15061,397 @@
+					"condition": "max_cards_in_hand",
+					"condition_amount": 0,
+					"effect_type": "powerup",
+					"amount": 4
+				}
+			]
+		}
+	},
+	{
+		"id": "merkava_irampage",
+		"type": "special",
+		"display_name": "I, Rampage",
+		"force_cost": 0,
+		"gauge_cost": 0,
+		"range_min": 1,
+		"range_max": 3,
+		"speed": 4,
+		"power": 4,
+		"armor": 0,
+		"guard": 0,
+		"effects": [
+			{
+				"timing": "before",
+				"effect_type": "advance",
+				"amount": 3
+			},
+			{
+				"timing": "hit",
+				"effect_type": "choice",
+				"choice": [
+					{ "effect_type": "push", "amount": 1 },
+					{ "effect_type": "pull", "amount": 1 }
+				]
+			},
+			{
+				"timing": "after",
+				"effect_type": "choice",
+				"choice": [
+					{ "effect_type": "return_attack_to_top_of_deck" },
+					{ "effect_type": "pass" }
+				]
+			}
+		],
+		"boost": {
+			"boost_type": "continuous",
+			"force_cost": 0,
+			"cancel_cost": -1,
+			"display_name": "Deflection",
+			"effects": [
+				{
+					"timing": "during_strike",
+					"effect_type": "powerup",
+					"amount": 1
+				},
+				{
+					"timing": "hit",
+					"effect_type": "armorup_current_power"
+				}
+			]
+		}
+	},
+	{
+		"id": "merkava_iagitate",
+		"type": "special",
+		"display_name": "I, Agitate",
+		"force_cost": 0,
+		"gauge_cost": 0,
+		"range_min": 1,
+		"range_max": 3,
+		"speed": 3,
+		"power": 5,
+		"armor": 0,
+		"guard": 5,
+		"effects": [
+			{
+				"timing": "hit",
+				"effect_type": "reveal_topdeck",
+				"reveal_to_both": true,
+				"and": {
+					"effect_type": "set_strike_x",
+					"source": "top_deck_power",
+					"and": {
+						"effect_type": "choice_altered_values",
+						"choice": [
+							{ "effect_type": "push", "amount": "strike_x" },
+							{ "effect_type": "pull", "amount": "strike_x" }
+						]
+					}
+				}
+			}
+		],
+		"boost": {
+			"boost_type": "continuous",
+			"force_cost": 0,
+			"cancel_cost": -1,
+			"display_name": "Lie In Wait",
+			"effects": [
+				{
+					"timing": "during_strike",
+					"effect_type": "guardup",
+					"amount": 3
+				},
+				{
+					"timing": "discarded",
+					"effect_type": "draw",
+					"amount": 2
+				}
+			]
+		}
+	},
+	{
+		"NOTE": "MERKAVA CARDS END",
+		"id": "merkava_ipersistentlycling",
+		"type": "special",
+		"display_name": "I, Persistently Cling",
+		"force_cost": 0,
+		"gauge_cost": 0,
+		"range_min": 0,
+		"range_max": 0,
+		"speed": 1,
+		"power": 7,
+		"armor": 0,
+		"guard": 5,
+		"effects": [
+			{
+				"timing": "set_strike",
+				"condition": "was_not_wild_swing",
+				"effect_type": "place_buddy_in_any_space",
+				"optional": true,
+				"buddy_name": "your attack"
+			},
+			{
+				"timing": "set_strike",
+				"condition": "buddy_in_play",
+				"condition_detail": "I, Persistently Cling",
+				"effect_type": "reveal_strike",
+				"and": {
+					"effect_type": "calculate_range_from_buddy"
+				}
+			},
+			{
+				"timing": "after",
+				"condition": "not_hit_opponent",
+				"effect_type": "return_attack_to_hand",
```

Comment:
Does cleanup still happen in this case? If not, need to remove the buddy here too

Reply:
Yup, active_strike still remembers the card when it gets removed